abs.yaml

```diff
@@ -0,0 +1,2 @@
+channels: 
+  cbouss: cairo
```

Comment:
Is it possible to remove this before integrating the package?

Reply:
Yes I'll remove it before merging. I needed it to work on other packages using this.